sponnet/pipeline.libsonnet

```diff
@@ -63,26 +69,23 @@
   },
 
   // notifications
-  // TODO: refactor level to include stage level.
 
-  notification(address, type, conditions, when, message):: {
+  notification:: {
     withAddress(address):: self + { address: address },
-    level: 'pipeline',
+    withCC(address):: self + { address: address },
```

Comment:
I address used by all notification types, not just email?

Reply:
Yes. `address` field is used for hipchat room, slack room, sms phone number, email address, etc.
Only email has an additional field available to it (`cc`) and have just pushed the naming fix for that now.
I hadn't bothered to split out `notification` into different types but can do. Might be more useful when the Github stuff comes along.